MaterialLib/MPL/Component.h

```diff
@@ -0,0 +1,62 @@
+/**
+ * \author Norbert Grunwald
+ * \date   Sep 7, 2017
+ *
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+#pragma once
+
+#include "BaseLib/ConfigTree.h"
+#include "Property.h"
```

Comment:
Unfortunately not, because the implementation of `value<T>()` etc. requires access to Property members.
I split the Enums.h in PropertyType.h and VariableType.h though.